src/system/sys_profiler.cc

```diff
@@ -36,11 +36,23 @@
 #include "sys_profiler.hh"
 #include "system/system.hh"
 #include <boost/format.hpp>
+#include <iostream>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/json_parser.hpp>
 
 #include "system/file_path.hh"
+#include "mpi.h"
+#include "timer_data.hh"
 
 
-#include "mpi.h"
+using boost::property_tree::ptree;
```

Comment:
Rater use name space renaming:
e.g. namespace boost_ptree =boost::property_tree::ptree;
or use full names if it is used on few places. This is better for readability.